web-build/Dockerfile.spx

```
@@ -0,0 +1,30 @@
+#ddev-generated
+RUN <<EOS
+set -e
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get install -y -o Dpkg::Options::="--force-confnew" --no-install-recommends --no-install-suggests \
+  build-essential make autoconf libc-dev pkg-config php-pear php${DDEV_PHP_VERSION}-dev zlib1g-dev
+EOS
+
+WORKDIR /tmp/php-spx
```

Hadolint (2.12.0)

[error] Invalid instruction order. Dockerfile must begin with `FROM`, `ARG` or comment.

(DL3061)